src/func.cc

```diff
@@ -499,6 +499,7 @@ static void ShellFuncImpl(const string& shell,
                           const string& cmd,
                           const Loc& loc,
                           string* s,
+                          int& returnCode,
```

Comment:
nit: Google style is to use const references (`const int& foo`) for input arguments and pointers (`int* foo`) for output arguments. That makes it more visible at the call site which argument are input and which are output.

Reply:
Done.